---
title: 移動の処理済み例外をデバッグします。
description: 移動の処理済み例外をデバッグします。
ms.assetid: f6288286-b57d-484e-b08a-79b2b84b6ea9
keywords:
- 移動の処理済み例外をデバッグします。
- ターゲットに移動して処理される例外のデバッグを制御します。
- 例外、移動処理の例外のデバッグ
ms.date: 05/23/2017
ms.localizationpriority: medium
ms.openlocfilehash: 23cf934e8723b2aedd977d0ddfdc22b88f8eda42
ms.sourcegitcommit: a33b7978e22d5bb9f65ca7056f955319049a2e4c
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 01/31/2019
ms.locfileid: "56558463"
---
# <a name="debug--go-handled-exception"></a>デバッグ |Go は、例外を処理


## <span id="ddk_debug_go_handled_exception_dbg"></span><span id="DDK_DEBUG_GO_HANDLED_EXCEPTION_DBG"></span>


クリックして**処理例外を参照してください**上、**デバッグ**] メニューの [ターゲットの実行を再開して、処理済みとしては、現在の例外を処理します。

### <a name="span-idadditionalinformationspanspan-idadditionalinformationspanadditional-information"></a><span id="additional_information"></span><span id="ADDITIONAL_INFORMATION"></span>追加情報

このアクションの影響の詳細については、このコマンドでは、およびプログラムの実行を制御する他の方法の他のメソッドを参照してください[ターゲットを制御する](controlling-the-target.md)します。 例外とその他のイベントの詳細については、[を制御する例外とイベント](controlling-exceptions-and-events.md)を参照してください。

 

 





